### PR TITLE
Unit test testing large batch exports

### LINKED ISSUE #1202: Integration tests - test large / split batches
The integration tests currently don't have coverage over large batches / split batches.

### PR DESCRIPTION
**Pull request by chaodaiG**
Fixes #1202

### CODE REVIEW

File: internal/export/worker_test.go [hunk 1]
```diff
@@ -148,220 +148,239 @@ func getKey(t *testing.T) []byte {
...
-	}
-	server := Server{
-		config: &config,
-		env:    serverenv.New(ctx, serverenv.WithDatabase(testDB)),
+	tests := []struct {
+		name  string
+		count int
+	}{
+		{
+			"base case",
```
Comment (sethvargo) [inline internal/export/worker_test.go:156]: Nit: don't use spaces in test case names

**Review by sethvargo — APPROVED** [review]

File: internal/export/worker_test.go (hunk 1 above)
Comment (chaodaiG) [reply internal/export/worker_test.go]: Done. Please add another lgtm

**Pull request merged**